smoketest/harness/harness.go

```diff
@@ -659,26 +659,26 @@ func (h *Harness) WaitAndAssertOnCallUsers(serviceID string, userIDs ...string)
 	getUsers := func() []string {
 		var result struct {
 			Service struct {
-				OnCall []struct {
-					UserID   string `json:"user_id"`
-					UserName string `json:"user_name"`
-				} `json:"on_call_users"`
+				OnCallUsers []struct {
+					UserID   string `json:"userID"`
```

Comment:
We can drop all these JSON tags, since they now match the field names (unmarshaling is already case-insensitive).